signer/src/transaction_signer.rs

```diff
@@ -413,6 +506,70 @@ mod tests {
         }
     }
 
+    struct EventLoopHarness {
+        event_loop: EventLoop,
+        notification_tx: tokio::sync::watch::Sender<()>,
+        storage: storage::in_memory::SharedStore,
+    }
+
+    impl EventLoopHarness {
+        fn create<Rng: rand::RngCore + rand::CryptoRng>(
+            rng: &mut Rng,
```

Comment:
nit: oh can we change the associated type to `R`, since it shadows the `rand::Rng` trait?

Reply:
Done